db/migrate/20250327130155_remove_default_billing_entity_sequential_id_on_invoices.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class RemoveDefaultBillingEntitySequentialIdOnInvoices < ActiveRecord::Migration[7.2]
+  disable_ddl_transaction!
+
+  def change
```

Comment:
ensure we always have an invoice with MAX billing_sequential_id assigned instead of nil if the organization is on per_customer.